pkgs/development/compilers/dtc/default.nix

```diff
@@ -22,7 +22,7 @@ stdenv.mkDerivation rec {
   makeFlags = [ "PYTHON=python" ];
   installFlags = [ "INSTALL=install" "PREFIX=$(out)" "SETUP_PREFIX=$(out)" ];
 
-  doCheck = true;
+  doCheck = !stdenv.isDarwin;
```

Comment:
Please add a comment why you disabled tests.

Reply:
done